src/lib/components/Toggle/Toggle.js

```diff
@@ -6,42 +6,36 @@
  * under the terms of the MIT License; see LICENSE file for more details.
  */
 
+import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import { Checkbox } from 'semantic-ui-react';
 import Overridable from 'react-overridable';
-import _get from 'lodash/get';
-import { Card } from 'semantic-ui-react';
-import PropTypes from 'prop-types';
+import { Card, Checkbox } from 'semantic-ui-react';
 
 class ToggleComponent extends Component {
-
   _isChecked = (userSelectionFilters) => {
-    const isFilterActive = userSelectionFilters.filter(
-      (filter) => filter[0] === this.props.filterValue[0]
-    ).length > 0
-    return isFilterActive
+    const isFilterActive =
+      userSelectionFilters.filter(
+        (filter) => filter[0] === this.props.filterValue[0]
+      ).length > 0;
+    return isFilterActive;
   };
 
   onToggleClicked = () => {
     this.props.updateQueryFilters(this.props.filterValue);
   };
 
   render() {
-    const {
-      userSelectionFilters,
-      overridableId,
-      title,
-      label
-    } = this.props;
+    const { userSelectionFilters, overridableId, title, label } = this.props;
     var isChecked = this._isChecked(userSelectionFilters);
-    const onToggleClicked = this.onToggleClicked
-    debugger
+    const onToggleClicked = this.onToggleClicked;
+    debugger;
```

Comment:
leftover? Was this merged before ????

Reply:
oh wow it looks like yes...are you opening the Developer Tools when working on RDM? ping @Glignos :) :D

Reply:
@ntarocco Oh well, that's on me. :)

Reply:
fixed